.gitignore

```diff
@@ -245,6 +245,24 @@ tags
 /test/memory/shannon/configVarFile.log
 
 /test/modules/gbt/printchplenv.good
+/test/modules/standard/FileSystem/fsouza/chdir/destination
+/test/modules/standard/FileSystem/fsouza/chown/file.txt
+/test/modules/standard/FileSystem/fsouza/isdir/my_dir
+/test/modules/standard/FileSystem/fsouza/isdir/my_file
+/test/modules/standard/FileSystem/fsouza/isfile/my_dir
+/test/modules/standard/FileSystem/fsouza/isfile/my_file
+/test/modules/standard/FileSystem/lydia/chmod/dir
+/test/modules/standard/FileSystem/lydia/chmod/file
+/test/modules/standard/FileSystem/lydia/exists/broken_link
+/test/modules/standard/FileSystem/lydia/exists/dir
+/test/modules/standard/FileSystem/lydia/exists/dir_link
+/test/modules/standard/FileSystem/lydia/exists/file
+/test/modules/standard/FileSystem/lydia/exists/file_link
+/test/modules/standard/FileSystem/lydia/exists/going_away
+/test/modules/standard/FileSystem/lydia/isLink/my_dir
+/test/modules/standard/FileSystem/lydia/isLink/my_file
+/test/modules/standard/FileSystem/lydia/isLink/my_link_dir
+/test/modules/standard/FileSystem/lydia/isLink/my_link_file
```

Comment:
This is a result of moving the directory from test/io way back when.  If you'll look a couple of lines above, most of these were originally blocked, but I accidentally missed updating the .gitignore when I moved them.  If you would remove the old lines (since they won't be useful again), that would be great.